instrumentation/servlet/servlet-3.0/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/servlet/v3_0/nowrapping/Servlet30AndFilterInstrumentation.java

```diff
@@ -130,13 +124,24 @@ public static boolean start(
         // skip execution of the user code
         return true;
       }
+
```

Comment:
We defer putting these in the instrumentation context until after `evaluateRequestHeaders` is called for two reasons:
1. If `evaluateRequestHeaders` returns `true`, these objects don't need to be in the instrumentation context because the request body will never be read
2. The header map needs to be passed along with the `Span` now so that the call to `evaluateRequestBody` can have access to the HTTP headers to make intelligent choices about whether the request should be blocked based on the value of the headers and the body, not just the body.